package/MDAnalysis/core/universe.py

```diff
@@ -387,11 +387,11 @@ def load_new(self, filename, format=None, in_memory=False, **kwargs):
                                  trj_n_atoms=self.trajectory.n_atoms))
 
         if in_memory:
-            self.transfer_to_memory(kwargs.get("in_memory_frame_interval", 1))
+            self.transfer_to_memory(step=kwargs.get("in_memory_frame_interval", 1))
```

Comment:
Yes, I was thinking about that but then I didn't do it right away because I thought the issue limits to only the `transfer_to_memory` method.